### ISSUE TITLE
Brewfest carts are misplaced

### ISSUE THREAD

**Issue by jamadaha** [azerothcore/azerothcore-wotlk#8025]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour
Original report: https://github.com/chromiecraft/chromiecraft/issues/1811

At the Brewfest camp in Dun Morogh, there is a tent that has Gordok Brew vendors. The tent is presently half-blocked by the Brewfest keg cart which has spawned ontop of the tent, blocking access to the Gordok Brew Apprentice who sells items. 

Testers note:
Misplaced carts:
GUID - 66861:
![image](https://user-images.githubusercontent.com/53914190/134233917-aba864af-8646-4ef8-ab6d-06b7b05f2588.png)
GUID - 66862:
![image](https://user-images.githubusercontent.com/53914190/134233953-92776ff2-3cde-4127-b721-250403856861.png)

### Expected Blizzlike Behaviour

The cart should be placed elsewhere or removed entirely so that one can access the vendor. 

Testers note:
This is not controversial. (I hope)

### Source

N/A

### Steps to reproduce the problem

1. ``` .go o 66861 ``` / ``` .go o 66862 ```

### Extra Notes

https://imgur.com/c9PyKTK a picture of the issue.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/7e2e6f8ee85f82f57a3c6e0f6f645b4c42292e7a

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by syssneck** [comment]
These wagons are there before the event (when they set up that camp), but should despawn, when the event starts, right?
Here we see the carts before the event started: https://youtu.be/fAEyXQEC2NM?t=37

**Comment by gurveleg** [comment]
Yes, They are part of the setting up the event before brewfest starts, like Darkmoon Faire has, and should be despawned once event started.

**Comment by Stump876** [comment]
They're gone now. the despawn appears to be delayed for a day or two.
![image](https://user-images.githubusercontent.com/68164076/134281792-b95fa1ed-0374-4d51-882a-deb8253005e8.png)
![image](https://user-images.githubusercontent.com/68164076/134281902-272195f7-933e-4f44-9f4e-4bb3b39d7602.png)

**Comment by UltraNix** [comment]
Duplicate of #25550
